src/view/payment-request.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import { StyleSheet, View } from 'react-native';
+import { observer } from 'mobx-react';
+import PropTypes from 'prop-types';
+import Background from '../component/background';
+import { NamedField } from '../component/field';
+import { Header, Title } from '../component/header';
+import Text from '../component/text';
+import { BackButton, CopyButton, SmallButton } from '../component/button';
+import { BalanceLabel } from '../component/label';
+import Card from '../component/card';
+import QRCode from '../component/qrcode';
+import { colors } from '../component/style';
+
+const styles = StyleSheet.create({
+  header: {
+    minWidth: 500,
+    width: '100%',
+    alignSelf: 'center',
+  },
```

Comment:
You can actually remove the `header` stylesheet completely and it won't change anything :D